openhexa/sdk/pipelines/runtime.py

```diff
@@ -19,6 +22,47 @@ def import_pipeline(pipeline_dir_path: str):
     return pipeline
 
 
+def get_pipeline_specs(pipeline_dir_path) -> typing.Tuple[PipelineSpecs, typing.List[PipelineParameterSpecs]]:
+    pipeline_dir = os.path.abspath(pipeline_dir_path)
+    sys.path.append(pipeline_dir)
+    pipeline_node = None
+
+    with open(pipeline_dir_path / Path("pipeline.py")) as f:
+        tree = ast.parse(f.read())
+        for node in ast.walk(tree):
+            if isinstance(node, ast.FunctionDef) and [
+                decorator for decorator in node.decorator_list if decorator.func and decorator.func.id == "pipeline"
+            ]:
+                pipeline_node = node
+                break
+
+    if pipeline_node:
+        pipeline_decorator = [
+            decorator for decorator in pipeline_node.decorator_list if decorator.func.id == "pipeline"
+        ]
+        pipeline_args = {}
+        for keyword in pipeline_decorator[0].keywords:
+            pipeline_args[keyword.arg] = (
+                keyword.value.value if isinstance(keyword.value, ast.Constant) else keyword.value.id
+            )
+
+        pipeline_specs = PipelineSpecs(code=pipeline_node.name, **pipeline_args)
+
+        param_decorators = [decorator for decorator in pipeline_node.decorator_list if decorator.func.id == "parameter"]
```

Comment:
Same remark than above (make sure that it is an actual `openhexa.sdk.parameter` function.